README.md

```diff
@@ -2,95 +2,71 @@
 
 # GDX Agreements Tracker
 
+<mark style="background-color: #fff; padding: 10px;">NOTE: To host this application in docker for local development, please see file here:</mark> [Docker Hosting GDX Agreements Tracker](/documentation/README.md)
+
+Below is the current preferred method for development. (ao. 03/25/2022)<br><br>
```

Comment:
This might seem a bit picky, but not my preferred method of date
```suggestion
Below is the current preferred method for development. (ao. Mar. 25, 2022)
```
```suggestion
Below is the current preferred method for development. (ao. 2022-03-25)
```